### PR TITLE
Anomaly detection doc

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

File: using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Anomaly Detection
+
+{% hint style="info" %}
+Anomaly Detection is currently in beta. Please read the documentation carefully.
+{% endhint %}
+
+Anomaly Detection is a governance tool which detects when cloud services significantly deviate from their projected spend. This feature, as well as [Forecasting](anomaly-detection.md#forecasting), are powered by the Anomaly Detection Container, which is enabled in Kubecost by default. Kubecost samples spend data from the last seven days to determine of a service's spend has become anomalous.
```
Comment (CampfireRemnants) [inline using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md:7]: It's probably better called the "modeling" container, as it's the home for all varieties of data-science/ML/etc. driven stuff.

File: using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+{% hint style="info" %}
+Anomaly Detection is currently in beta. Please read the documentation carefully.
+{% endhint %}
+
+Anomaly Detection is a governance tool which detects when cloud services significantly deviate from their projected spend. This feature, as well as [Forecasting](anomaly-detection.md#forecasting), are powered by the Anomaly Detection Container, which is enabled in Kubecost by default. Kubecost samples spend data from the last seven days to determine of a service's spend has become anomalous.
+
+![Anomaly detection](/images/anomalydetection.png)
+
+## Forecasting
```
Comment (CampfireRemnants) [inline using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md:11]: Should we include details about the underlying models? E.g. Forecasting is powered by ARIMA?

**Review by CampfireRemnants — COMMENTED** [review]
Just a couple clarifying remarks re: forecasting and anomaly detection.

File: using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md (hunk 2 above)
Comment (bstuder99) [reply using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md:11]: I have not heard about any of that. Is there an internal doc about it?

Comment (CampfireRemnants) [reply using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md:11]: I mean, there's my planning doc, I guess? https://link.excalidraw.com/l/ABLQ24dkKai/5P9UVgEeiYn But there's not anything else I can think of at the moment.

I can tell you that Forecasting uses an ARIMA model, and Anomaly detection uses Z-Score Fences, and I'm happy to break down more technical detail if that's relevant.

Comment (bstuder99) [reply using-kubecost/navigating-the-kubecost-ui/anomaly-detection.md:11]: There's a lot to unpack in this chart, at this current time I'm not sure how crucial this information will be for users upon release. 've heard some conflicting things about how forecasting/anomaly detection are powered so until I can get it cleared up, I don't think technical specifics are needed.

**Pull request merged**